src/store/selectors/getQueryParamsObjectFromString.js

```diff
@@ -0,0 +1,10 @@
+const getQueryParamsObjectFromString = (queryParamString) => {
+  let obj = {}
+  const params = new URLSearchParams(queryParamString)
+  for (const [key, value] of params) {
+    obj[key] = value
+  }
+  return obj
+}
+
+export default getQueryParamsObjectFromString
```

Comment:
Similar functions live in selectors however it feels like more of a util since it doesn’t get anything from the redux store. Should this function live here or somewhere else?